src/main/java/bio/terra/cli/command/workspace/ConfigureAws.java

```diff
@@ -1,175 +1,63 @@
 package bio.terra.cli.command.workspace;
 
+import static bio.terra.cli.utils.AwsConfiguration.AWS_CONFIG_FILE_ENVIRONMENT_VARIABLE;
+import static bio.terra.cli.utils.AwsConfiguration.DEFAULT_AWS_VAULT_PATH;
+import static bio.terra.cli.utils.AwsConfiguration.DEFAULT_CACHE_WITH_AWS_VAULT;
+import static bio.terra.cli.utils.AwsConfiguration.DEFAULT_TERRA_PATH;
+
 import bio.terra.cli.businessobject.Context;
-import bio.terra.cli.businessobject.Resource;
-import bio.terra.cli.businessobject.Workspace;
 import bio.terra.cli.command.shared.WsmBaseCommand;
 import bio.terra.cli.command.shared.options.WorkspaceOverride;
-import bio.terra.cli.exception.SystemException;
-import bio.terra.cli.exception.UserActionableException;
 import bio.terra.cli.utils.AwsConfiguration;
-import bio.terra.cli.utils.FileUtils;
-import com.google.common.annotations.VisibleForTesting;
-import java.io.IOException;
 import java.nio.file.Path;
-import java.util.List;
 import picocli.CommandLine;
 
 /** This class corresponds to the third-level "terra workspace configure-aws" command. */
 @CommandLine.Command(
     name = "configure-aws",
     description = "Generate an AWS configuration file for a workspace.")
 public class ConfigureAws extends WsmBaseCommand {
-  private static final String AWS_CONTEXT_SUBDIRECTORY_NAME = "aws";
-  public static final String DEFAULT_TERRA_PATH = "/usr/local/bin/terra";
-  public static final String DEFAULT_AWS_VAULT_PATH = "/usr/local/bin/aws-vault";
-  public static final String AWS_CONFIG_FILE_ENVIRONMENT_VARIABLE = "AWS_CONFIG_FILE";
-
   @CommandLine.Mixin WorkspaceOverride workspaceOption;
 
   @CommandLine.Option(
       names = "--cache-with-aws-vault",
-      defaultValue = "false",
-      description = "Use aws-vault for caching")
+      description = "Use aws-vault for caching. Default - " + DEFAULT_CACHE_WITH_AWS_VAULT)
```

Comment:
- Builder default vs option default values: I would say the builder default is the safest location for default values in general if we want to ensure a valid value for a field, as long as the default is widely applicable. 
here - terra and aws paths for example are set to default values where they are installed. 
If the cli option is used in multiple places where a different default is needed, then agree - setting default in option is better. 
example - if this is part of a script that installs terra in a different location, option default would be better. 
Since the option is not likley to be shared, I chose to put the defaults in builder. I am not against changing this to option defaults if its more usable.
---
 - option cache-with-aws-vault: default value is currently set to false (its false if its not provided). If its changed to true, then this wont work. 
 thanks for pointing this. fixing

Reply:
more info - see that workspace is required, so its validated in Builder (just that its provided). If defaults are only at the option, all values may beed validation.